src/styles/colors.tsx

```diff
@@ -1,53 +1,44 @@
 // Designer Created Figma Colors
```

Comment:
the key change is in this file. to make it easier to review, i'll group changes in targetted PRs. the first part of the enum will eventually all be axed 🪓